app/models/groupe_instructeur.rb

```diff
@@ -95,7 +103,7 @@ def non_unique_rule?
   def groups_with_same_rule
     return if routing_rule.nil?
     other_groupe_instructeurs
-      .filter { |gi| !gi.routing_rule.nil? && gi.routing_rule.right != empty && gi.routing_rule == routing_rule }
+      .filter { |gi| !gi.routing_rule.nil? && gi.routing_rule == routing_rule }
```

Comment:
chipotage, utilisation de `present?` pour éviter la négation

```suggestion
      .filter { _1.routing_rule.present? }
      .filter { _1.routing_rule == routing_rule }
```